upstream/CVE-2021-25749.json

```
@@ -0,0 +1,89 @@
+{
+  "id": "CVE-2021-25749",
+  "modified": "2022-09-01T21:02:01Z",
+  "published": "2022-09-01T21:02:01Z",
+  "summary": "`runAsNonRoot` logic bypass for Windows containers",
+  "details": "Windows workloads can run as ContainerAdministrator even when those workloads set the runAsNonRoot option to true.",
+  "affected": [
+    {
+      "package": {
+        "ecosystem": "kubernetes",
+        "name": "k8s.io/kubelet"
+      },
+      "severity": [
+        {
+          "type": "CVSS_V3",
+          "score": "CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:H/I:H/A:H"
+        }
+      ],
+      "ranges": [
+        {
+          "events": [
+            {
+              "introduced": "1.22.0"
+            },
+            {
+              "fixed": "1.22.14"
```

```suggestion
              "introduced": "1.20.0"
            },
            {
              "fixed": "1.22.14"
```

from issue:
> kubelet v1.20 - v1.21
kubelet v1.22.0 - v1.22.13

---

fixed in https://github.com/aquasecurity/vuln-list-k8s/pull/88/commits/83b82902255dbffab351734a73330aa8c370352e